openshift/openshift.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
```

Comment:
make...file...make...file...make..file...

I know, you guys hate me for those makefiles everywhere ;)

Reply:
Make? But Bash is Turing complete, isn't it?!

Reply:
I'll think about it. But all other scripts in RadarGun are Bash style. So some people will object that other scripts are Bash and some will object that running this script would be easier if it was a makefile :)

Reply:
I'd consider using makefile only for building Docker image as it gives some hint there some dependencies between tasks and Dockerfile cannot be used standalone (as I was confused in comment above).